crates/oxc_minifier/README.md

```diff
@@ -29,6 +29,8 @@ The compressor is responsible for rewriting statements and expressions for minim
   - Examples that breaks this assumption: `(() => { console.log(v); let v; })()`
 - `with` statement is not used
   - Examples that breaks this assumption: `with (Math) { console.log(PI); }`
+- `.toString()`, `.valueOf()`, `[Symbol.toPrimitive]()` are side-effect free
+  - Examples that breaks this assumption: `{ toString() { console.log('sideeffect') } }`
```

Comment:
uglify, terser, closure compiler makes this assumption. All of them does not say anything about `Symbol.toPrimitive`, but it doesn't make sense without it and I assume they do so.